src/framework/global/thirdparty/kors_logger/src/logdefdest.cpp

```diff
@@ -110,12 +110,24 @@ void ConsoleLogDest::write(const LogMsg& logMsg)
     std::string msg;
     msg.reserve(100);
     msg.append(colorCode(logMsg.color));
-    msg.append(m_layout.output(logMsg));
+    std::string log = m_layout.output(logMsg);
+    msg.append(log);
     msg.append("\033[0m");
 #ifdef _WIN32
-    std::wstring temp = std::wstring(msg.begin(), msg.end());
-    OutputDebugString(temp.c_str());
+    std::wstring str = std::wstring(msg.begin(), msg.end());
+    size_t preview = 0;
+    for (size_t i = 0; i < str.size(); ++i) {
+        if (str.at(i) == L'\n') {
+            str[i] = L'\0';
+            OutputDebugString(&str[preview]);
+            OutputDebugString(L"\n");
+            preview = i + 1;
+        }
+    }
+
+    OutputDebugString(&str[preview]);
     OutputDebugString(L"\n");
+
 #else
     std::cout << msg << std::endl;
```

Comment:
shouldn't this rather be `std::stderr`?

Reply:
A variety of messages can be output here, not just “errors”. I don’t know the reasons why it is worth dividing conclusions into errors and non-errors.

Reply:
Because they intermix with 'regular' output?

Reply:
See #13304

Reply:
thank you, changed to std::cerr

Reply:
The "error stream" is a bad name for what is really the "message stream".

The output stream should only be used for requested output.

Reply:
IOW you agree with me, right?

Reply:
Yes